symfony/framework-bundle/5.1/src/Kernel.php

```diff
@@ -15,14 +16,41 @@ protected function configureContainer(ContainerConfigurator $container): void
     {
         $container->import('../config/{packages}/*.yaml');
         $container->import('../config/{packages}/'.$this->environment.'/*.yaml');
-        $container->import('../config/{services}.yaml');
-        $container->import('../config/{services}_'.$this->environment.'.yaml');
+
+        // Put parameters here that don't need to change on each machine where the app is deployed
+        // https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
+        //$parameters = $container->parameters();
+        //$parameters->set(...);
+
+        $services = $container->services();
+
+        // makes classes in src/ available to be used as services
+        $services
+            ->load('App\\', './*')
+            ->exclude('./{DependencyInjection,Entity,Migrations,Tests,Kernel.php}');
+
+        // controllers are imported separately to make sure services can be injected
+        // as action arguments even if you don't extend any base controller class
+        $services
+            ->load('App\\Controller\\', './Controller')
+            ->tag('controller.service_arguments');
+
+        // add more service definitions when explicit configuration is needed
+        // please note that last definitions always *replace* previous ones
+        //$services->set(...);
     }
 
     protected function configureRoutes(RoutingConfigurator $routes): void
     {
         $routes->import('../config/{routes}/'.$this->environment.'/*.yaml');
         $routes->import('../config/{routes}/*.yaml');
-        $routes->import('../config/{routes}.yaml');
+
+        $routes->add('hello', '/hello')
```

Comment:
Should we add this to the recipe? If no routes are defined you are being greeted by the new welcome page, right?

Reply:
PR is draft for a reason ;)

Reply:
Well, the welcome page is shown in the root path `/` only, so there's no conflict, although I'm not sure why we might need this route here. Most of the time we don't worry about the kernel file, I wouldn't want to remove this line every time I create a new project or forget where it came from.